source/val/validate_image.cpp

```diff
@@ -1903,6 +1911,15 @@ spv_result_t ValidateImageQueryLod(ValidationState_t& _,
            << "Expected Coordinate to have at least " << min_coord_size
            << " components, but given only " << actual_coord_size;
   }
+
+  const auto target_env = _.context()->target_env;
```

Comment:
This seems off. The `SampledImage` operand must be an `OpTypeSampledImage`. The validator check that `OpSampledImage` must have `Sampled` equal to 1 for Vulkan, but does not check that for `OpTypeSampledImage`. I would suggest removing this check and adding one in `ValidateTypeSampleImage()`.

Reply:
Done.
I'll check agianst Vulkan CTS master branch.